lib/system/freertos/sys.h

```diff
@@ -19,8 +19,6 @@
 
 #include <metal/errno.h>
 
-#include "./@PROJECT_MACHINE@/sys.h"
```

Comment:
Could you explain why you remove this?
This probably break the build with the template or another machine

Reply:
Hi @arnopo, your concern is valid. but that line forces us to duplicate sys.h interfaces for each platforms. Instead, can we move sys.h interfaces only into freertos/sys.h and then each platform's sys.c should implement those interfaces. This will reduce duplicate sys.h files for each platforms. For example, Only for Xilinx Platforms, we have about 6 duplicate sys.h files that contains same interfaces.

Reply:
I understand the redundancy, This is linked to the way the libmetal is designed. 
What we have to keep in mind here is that this will generate regression on projects that use the Libmetal  generic or Freertos environments. 
So Instead of removing this for all platforms you should make an exception for the Xilinx platforms